privacyidea/lib/tokens/yubikeytoken.py

```diff
@@ -372,6 +372,8 @@ def api_endpoint(cls, request, g):
             data['status'] = "NO_SUCH_CLIENT"
             data['h'] = ""
         elif otp and id and nonce:
+            options = {"g": g,
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -372,6 +372,8 @@ def api_endpoint(cls, request, g):
>>              data['status'] = "NO_SUCH_CLIENT"
>>              data['h'] = ""
>>          elif otp and id and nonce:
>> +            options = {"g": g,
>
> The options are not used anywhere!?

Yes, that's what I figured out, but now got a message that parameter g
in that function is unused.  I'll try to figure out who is calling with
what parameters...

For now, please disregard the pull request - I'll prepare another one
after release.

Jochen

-- 
The only problem with troubleshooting is that the trouble shoots back.